app/views/foreman_tasks/task_groups/recurring_logic_task_groups/_recurring_logic_task_group.html.erb

```diff
@@ -1,5 +1,5 @@
 <% recurring_logic = task_group.recurring_logic -%>
-<table class='table table-condensed'>
+<table class='<%= table_css_classes('table-condensed') %>'>
```

Comment:
I'm not sure if it makes sense here. The table is used just to get an aligned list of key-value pairs. Maybe using a  table for it wasn't the best choice in the first place, but... Anyway, if we need the tables to be consistent everywhere, I won't block this.

![image](https://user-images.githubusercontent.com/7326770/36670876-fedac864-1af9-11e8-9dce-f306f4f04eed.png)

Reply:
Agreed it doesn't make sense here.

Reply:
we have the same in host detail page in propertis/nics tabs, why would you prefer to be inconsistent here?

Reply:
I somehow missed it being in properties/nics table as well. It doesn't make much sense to me there either, but guess we should stay consistent with core.